packages/expo-video/src/VideoView.web.tsx

```diff
@@ -94,16 +95,37 @@ export const VideoView = forwardRef((props: { player?: VideoPlayer } & VideoView
     hasToSetupAudioContext.current = false;
   }
 
+  function fullscreenListener() {
+    if (document.fullscreenElement === videoRef.current) {
+      props.onFullscreenEnter?.();
+    } else {
+      props.onFullscreenExit?.();
+    }
+  }
+
+  function setupFullscreenListener() {
+    fullscreenChangeListener.current = fullscreenListener;
+    videoRef.current?.addEventListener('fullscreenchange', fullscreenChangeListener.current);
+  }
+
+  function cleanupFullscreenListener() {
+    if (fullscreenChangeListener.current) {
+      videoRef.current?.removeEventListener('fullscreenchange', fullscreenChangeListener.current);
```

Comment:
Can we unset the ref here?
```suggestion
      videoRef.current?.removeEventListener('fullscreenchange', fullscreenChangeListener.current);
      fullscreenChangeListener.current = null;
```

Reply:
You're right, I missed that. Added.